### PR TITLE
Set PVC accessMode based on source VM

### PR DESCRIPTION
**Pull request by machacekondra**
This PR set the access mode of the DataVolume's PVC based on the source
VM configuration. If the source VM disk is readonly we create read-only
PVC. If the source VM has migratable affinity set, we create
ReadWriteMany PVC.

Signed-off-by: Ondra Machacek <omachace@redhat.com>

### CODE REVIEW

File: pkg/providers/ovirt/mapper/mapper.go [hunk 1]
```diff
@@ -258,7 +275,7 @@ func (o *OvirtMapper) MapDataVolumes(targetVMName *string) (map[string]cdiv1.Dat
 				},
 				PVC: &corev1.PersistentVolumeClaimSpec{
 					AccessModes: []corev1.PersistentVolumeAccessMode{
-						corev1.ReadWriteOnce,
+						accessMode,
```
Comment (masayag) [inline pkg/providers/ovirt/mapper/mapper.go:278]: will this require changes to the functional tests setup to make sure PVs with proper access modes are available or the default PVs supports all types of access-modes ?

Comment (machacekondra) [reply pkg/providers/ovirt/mapper/mapper.go:278]: Actually, you can create `ReadWriteMany` PVC even on storageclass, which doesn't support RWX acccess mode. So it doesn't but I will add `rook-ceph` in other PR, so we have real test case, and also we may test multiple storage classes.

**Review by pkliczewski — APPROVED** [review]

**Review by jakub-dzon — APPROVED** [review]

**Review by masayag — APPROVED** [review]

**Comment by machacekondra** [comment]
Fixes: https://github.com/kubevirt/vm-import-operator/issues/267

**Pull request merged**